app/stac_api/apps.py

```diff
@@ -112,11 +112,13 @@ def custom_exception_handler(exc, context):
     response = exception_handler(exc, context)
 
     if response is not None:
+        extra = {"request": context['request']._request, "requestPayload": context['request'].data}  # pylint: disable=protected-access
+        logger.error(repr(exc), extra=extra)
```

Comment:
I would not log the exception here, the exception is handled. I would log `Response %s: %s % (status_code, response.data)`

Reply:
:+1: